projects/uitdatabank/docs/entry-api/events/status.md

```diff
@@ -1,8 +1,141 @@
+
+## The status property
+
+Every event automatically gets a `status` property after it is created.
+
+By default, it looks like this:
+
+```json
+{
+  "status": {
+    "type": "Available"
+  }
+}
+```
+
+The nested `type` property can either be `Available` (happening as scheduled), `Unavailable` (cancelled) or `TemporarilyUnavailable` (postponed). 
+
+The possible `type` values are intentionally worded in a generic way so that they match those of [the `status` property on places](../places/status.md), which makes it easier to query both events and places at the same time on their status property [using Search API](../../search-api/introduction.md).
+
+When the event has calendarType `single` or `multiple`, the objects inside its `subEvent` property will also automatically get the same `status` property. 
+
+For example on an event with multiple dates:
+
+```json
+{
+  "status": {
+    "type": "Available"
+  },
+  "calendarType": "multiple",
+  "subEvent": [
+    {
+      "startDate": "2023-01-18T17:30:00+01:00",
+      "endDate": "2023-01-18T23:00:00+01:00",
+      "status": {
+        "type": "Available"
+      }
+    },
+    {
+      "startDate": "2023-01-19T17:30:00+01:00",
+      "endDate": "2023-01-19T23:00:00+01:00",
+      "status": {
+        "type": "Available"
+      }
+    }
+  ]
+}
+```
+
+## Cancelling an event
+
+### With calendarType "single" or "multiple"
+
+To cancel an event with calendarType `single` or `multiple`, update the `status.type` property of the `subEvent` objects that should be cancelled and set it to `Unavailable`.
+
+Optionally, you can include a `reason` property to explain why the event is cancelled on that date. The `reason` property must be an object with language codes like `name`.
+
+For example to cancel one date of multiple:
+
+```json
+{
+  "calendarType": "multiple",
+  "subEvent": [
+    {
+      "startDate": "2023-01-18T17:30:00+01:00",
+      "endDate": "2023-01-18T23:00:00+01:00",
+      "status": {
+        "type": "Available"
+      }
+    },
+    {
+      "startDate": "2023-01-19T17:30:00+01:00",
+      "endDate": "2023-01-19T23:00:00+01:00",
+      "status": {
+        "type": "Unavailable",
+        "reason": {
+          "nl": "Afgelast wegens COVID-19 maatregelen."
+        }
+      }
+    }
+  ]
+}
+```
+
+Note that you may omit the `status` property on the top level, as it will automatically be set based on the `status` of each `subEvent`. If you include it anyway, it will be ignored.
+
+The top-level `status` is determined as follows:
+
+* When at least one `subEvent` has its `status.type` set to `Available`, the top-level `status.type` will automatically be set to `Available` since there is at least one date that the event is still happening as scheduled. 
+* When all subEvents have their `status.type` set to `Unavailable`, the top-level `status.type` will also be set to `Unavailable` since this means that all dates are cancelled.
```

Comment:
```suggestion
* When all subEvents have their `status.type` set to `Unavailable`, the top-level `status.type` will also be set to `Unavailable` since this means that all dates are cancelled.
* When all subEvents have their `status.type` set to `temporarilyUnavailable`, the top-level `status.type` will also be set to `temporarilyUnavailable` since this means that all dates are postponed.
* When the subEvents have different statuses, but one or more subEvents have the `status.type` set to `temporarilyUnavailable`, and none to `Available`, the top-level `status.type` will automatically be set to `temporarilyUnavailable` since there is at least one date that the event still takes places, but on a later date.
```

Reply:
Tbh this becomes a lot of text, maybe a table makes it more clear:


| status.type subEvents | Derived status.type on top event level |
| :-- | :-- |
| At least one `Available` | `Available` |
| All `Unavailable` | `Unavailable` |
| All `temporarilyUnavailable` | `temporarilyUnavailable` |
| Mixed, but one or more `temporarilyUnavailable` and none `Available` | `temporarilyUnavailable` |

Reply:
I think this is correct as well and a bit shorter?

```suggestion
* Otherwise, when at least one `subEvent` has its `status.type` set to `TemporarilyAvailable`, the top-level `status.type` will automatically be set to `TemporarilyAvailable` since there is still at least one date that the event still takes place, but on a later date.
* Otherwise, the top-level `status.type` will automatically be set to `Unavailable` since every `subEvent` will have its `status.type` set to `Unavailable` which means that the event is cancelled completely.
```

Reply:
I have no preference.